drones/connection/prototype.py

```diff
@@ -0,0 +1,22 @@
+"""Basic drone connection and communication.
+
+Additional info.
+"""
+
+import logging
+
+
+class ConnectorPrototype:
+    def __init__(self):
+        self.log = logging.getLogger(__name__)  # Get global logger setup
+
+    def connect(self):
+        """Sample public function, which is accessible from outside packages and is entrypoint of the library"""
+
+        self.log.info("Connection established")
+        self.log.warning("Tracking errors")
+
+    def __track_errors(self):
+        """Sample private function, which is inaccessible from outside of the package"""
+
+        self.log.error("Sample log error")

```

Comment:
Regarding your import issues, I suggest creating an instance of `ConnectorPrototype` here e.g. `connector = ConnectorPrototype()`. Then, you will be able to change your import in `__init__.py` to `from drones.connection.prototype import connector`,